packages/@react-aria/virtualizer/src/VirtualizerItem.tsx

```diff
@@ -57,7 +57,7 @@ export function layoutInfoToStyle(layoutInfo: LayoutInfo, dir: Direction, parent
 
   let style: CSSProperties = {
     position: layoutInfo.isSticky ? 'sticky' : 'absolute',
-    overflow: 'hidden',
+    overflow: layoutInfo.overflow,
```

Comment:
Open to opinions here. @snowystinger and I discussed alternatives to using `!important` in the css to override the `overflow` and `contain` inline styles that VirtualizerItem provides to the item wrapping divs and settled on having `layoutInfo` contain that information. Now it is up to the layouts to override the default of `overflow: hidden` and `contain: size layout style paint` that we normally apply to the much like how `isSticky`/`width`/etc is applied.

An alternative idea was to have VirtualizerItem accept a user provided `style` (typically done in renderWrapper) and have those styles overwrite the styles sourced from `layoutInfoToStyle`

Reply:
Maybe rather than exposing the raw CSS properties, we could abstract this a bit? Like have a single property `allowOverflow` or something that when set automatically sets overflow and contain appropriately? Since layouts live in stately would be good to keep things a bit more abstracted.

Reply:
Just to make sure I'm on the same page, the proposed change here would be to support `layoutInfo.allowOverflow = true` instead of `layoutInfo.overflow`? And if `allowOverflow=true`, then have VirtualizerItem's `overflow` and `contain` be `overflow: visible` and `contain: 'size layout style'`